group_user_management/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright (c) 2024- Le Filament (https://le-filament.com)
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    signup_token = fields.Char(
+        copy=False,
+        groups="base.group_erp_manager, group_user_management.group_user_manager",
+        compute="_compute_token",
+        inverse="_inverse_token",
+    )
```

Comment:
```suggestion
    signup_token = fields.Char(
        groups="base.group_erp_manager, group_user_management.group_user_manager",
    )
```

when overloading fields, only redefine changed values.